neqo-transport/src/packet.rs

```diff
@@ -614,6 +614,20 @@ impl<'a> PublicPacket<'a> {
             Err(Error::DecryptError)
         }
     }
+
+    pub fn decode_version_negotiation(&self) -> Res<Vec<Version>> {
```

Comment:
let's call this something a little less about the implementation details and more about the result. Maybe `supported_versions()`? Or you tell me.